<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TNetwork</class>
 <widget class="QWidget" name="Gui::Pref::TNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="proxy_tab">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QGroupBox" name="proxy_group">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="title">
          <string>&amp;Enable proxy</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="proxy_hostname_edit"/>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="proxy_port_spin">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>proxy_port_spin</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_username_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="proxy_type_combo"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&amp;Type:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_type_combo</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="proxy_password_edit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Host:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_hostname_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="proxy_username_edit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pa&amp;ssword:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_password_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>label_2</zorder>
         <zorder>proxy_hostname_edit</zorder>
         <zorder>label_3</zorder>
         <zorder>proxy_port_spin</zorder>
         <zorder>label_4</zorder>
         <zorder>proxy_username_edit</zorder>
         <zorder>label_5</zorder>
         <zorder>proxy_password_edit</zorder>
         <zorder>label_6</zorder>
         <zorder>proxy_type_combo</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>proxy_group</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
